nixos/modules/system/boot/stage-1.nix

```diff
@@ -107,6 +110,22 @@ let
         copy_bin_and_libs $BIN
       done
 
+
+      # Filesystems using the "zfsutil" option do mount regardless of the
+      # mount.zfs(8) helper, but it is required to ensure that ZFS properties
+      # are used as mount options.
```

Comment:
Very minor nitpick, because sometimes I find myself looking through generated scripts like this: Could you move this comment inside the `optionalString` block?

Reply:
Of course.